### PR TITLE
feat: 在连接 kubernetes 时增加 helm 工具命令

### PR DESCRIPTION
**Pull request by get-set**
- 增加 helm 工具的 wrapper
- build 脚本中增加编译 helm wrapper 相关的命令
- Dockerfile 中加入 rawhelm 和 helm wrapper
- kubectl 和 helm 的 wrapper 公共部分进行了重构

### CODE REVIEW

**Comment by wojiushixiaobai** [comment]
- [ ] linx: jumpserver/jumpserver#8476

File: Dockerfile [hunk 1]
```diff
@@ -24,15 +24,15 @@ ENV GO111MODULE=on
 ENV GOOS=linux
 
 RUN wget https://download.jumpserver.org/public/kubectl-linux-${TARGETARCH}.tar.gz -O kubectl.tar.gz \
-    && tar -xzf kubectl.tar.gz \
+    && tar -xf kubectl.tar.gz \
     && chmod +x kubectl \
     && mv kubectl rawkubectl \
-    && wget https://get.helm.sh/helm-v3.9.0-linux-${TARGETARCH}.tar.gz -O helm.tar.gz \
-    && tar -xzf helm.tar.gz \
+    && wget https://download.jumpserver.org/public/helm-v3.9.0-linux-${TARGETARCH}.tar.gz -O helm.tar.gz \
```
Comment (get-set) [inline Dockerfile:30]: Thanks

**Review by LeeEirc — APPROVED** [review]

**Comment by LoveEachDay** [comment]
@LeeEirc When would this pr be released? Since the most recently release 2.23.4 doesn't contain this pr.

**Comment by LeeEirc** [comment]
> @LeeEirc When would this pr be released? Since the most recently release 2.23.4 doesn't contain this pr.
this pr will be released on v2.24.0

**Pull request merged**